File: JS-Advanced/JS-Advanced/L01SyntaxFunctions/Lab/sumNumbers.js
function sum(n, m){
    let start = Number(n);
    let end = Number(m);

    let sum = 0;

    for(let x = start; x <= end; x++){
        sum += x;
    }

    console.log(sum);
}

sum('1', '5');
sum('-8', '20');

File: Задачи/Fundamentals/ForLoop/SumNumbers/SumNumbers.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SumNumbers
{
    class SumNumbers
    {
        static void Main(string[] args)
        {
            int sum = 0;
            int number = 0;
            int n = int.Parse(Console.ReadLine());

            for(int i = 0; i < n; i++)
            {
                number = int.Parse(Console.ReadLine());
                sum += number;
            }

            Console.WriteLine(sum);
        }
    }
}
